caffe2/serialize/versions.h

```diff
@@ -6,7 +6,7 @@ namespace caffe2 {
 namespace serialize {
 
 constexpr uint64_t kMinSupportedFileFormatVersion = 0x1L;
-constexpr uint64_t kMaxSupportedFileFormatVersion = 0x6L;
+constexpr uint64_t kMaxSupportedFileFormatVersion = 0x7L;
```

Comment:
this doesn't feel right. @cccclai @gmagogsfm Seems like mobile bytecode version relies on this? if this is not needed, I would like to remove it.

Reply:
`kMaxSupportedFileFormatVersion` (global operator version) is different than `kProducedBytecodeVersion` (bytecode schema version). I don't see the bytecode schema will change with this diff, so there is no need to bump `kProducedBytecodeVersion`. @iseeyuan what do you think?